src/utils/geo.js

```diff
@@ -0,0 +1,201 @@
+import Cookies from 'js-cookie';
```

Comment:
Just a thought which doesn't need to be addressed now: how about creating another new repo for geo.js? ;) It's not really an `ad-product` and it also doesn't really fit to `ad-engine`. Poor module :-(